src/test/java/com/dabsquared/gitlabjenkins/webhook/build/MergeRequestBuildActionTest.java

```diff
@@ -107,6 +108,9 @@ public void build() throws IOException {
             verify(mockTrigger).onPost(pushHookArgumentCaptor.capture());
             assertThat(pushHookArgumentCaptor.getValue().getProject(), is(notNullValue()));
             assertThat(pushHookArgumentCaptor.getValue().getProject().getWebUrl(), is(notNullValue()));
+            assertThat(pushHookArgumentCaptor.getValue().getUser(), is(notNullValue()));
+            assertThat(pushHookArgumentCaptor.getValue().getUser().getName(), containsString("Administrator"));
+            assertThat(pushHookArgumentCaptor.getValue().getUser().getUsername(), containsString("root"));
```

Comment:
Actually this only checks for when the User's name contains "Administrator" and the User's username is "root" by design. But there should be no harm done by adding this code snippet.